homeassistant/components/media_player/kodi.py

```diff
@@ -28,31 +28,28 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     add_devices([
         KodiDevice(
             config.get('name', 'Kodi'),
-            config.get('url'),
+            config.get('host'),
```

Comment:
Can you sort out the configuration stuff in the `setup_platform` method instead of in the constructor? It's a better separation of concerns.